tastytaps/taps/models.py

```diff
@@ -30,3 +30,18 @@ class Price(models.Model):
 
     def __unicode__(self):
         return self.size
+
+
+class Taproom(models.Model):
+    name = models.CharField(max_length=200)
+    num_taps = models.PositiveSmallIntegerField(blank=True, null=True)
+    address = models.CharField(max_length=200, blank=True)
+
+    def __unicode__(self):
+        return self.name
+
+
+class Taps(models.Model):
+    beer = models.ForeignKey(Beer)
+    taproom = models.ForeignKey(Taproom, related_name='taps')
+    tap_number = models.PositiveSmallIntegerField(blank=True, null=True)
```

Comment:
My thought would be to keep it simple and require it. We can explore other use-cases later when we launch. This feels like something that could make it more complicated now when we may not need it. Maybe bars want an "On deck" list of beers that are coming up soon but aren't tapped yet. But once we have some users we could solicit feedback like that.